internal/framework/status/updater_test.go

```diff
@@ -508,3 +509,119 @@ var _ = Describe("Updater", func() {
 		})
 	})
 })
+
+func TestUpdateObjStatus(t *testing.T) {
```

Comment:
We don't usually mix gomega tests and go x-unit tests in the same test file. I would throw these in another test file.

Reply:
Removed test files because we use imported retrying function.